tests/phpunit/integration/Core/Authentication/Clients/OAuth_ClientTest.php

```diff
@@ -26,6 +27,7 @@
  * @group Authentication
  */
 class OAuth_ClientTest extends TestCase {
+	use Authentication_Trait;
 
 	const SITE_ID   = '12345678.apps.sitekit.withgoogle.com';
 	const CLIENT_ID = '12345678.apps.googleusercontent.com';
```

Comment:
These constants refer to values that are in the trait now. Let's get rid of these constants and update the methods on the trait to also return the fake credentials set in the filter. We don't care about the whole thing just the array set to the `web` key, so it returns an array with `client_id` and `client_secret`.

Then we can update the references in the test to use the client ID returned from the `fake_` methods rather than the class constant 👌

Reply:
Fixed